base-rec.html

```diff
@@ -10102,6 +10102,7 @@ <h5 id="dictionary-rtcrtpheaderextensioncapability-members">
             </section>
           </div>
         </section>
+        <div id="rtcencodeoptions"></div>
```

Comment:
it seems this can't be a `div`?
```
"file:/home/runner/work/webrtc-pc/webrtc-pc.w3c/webrtc.html":11238.830-11238.930: error: Element “div” not allowed as child of element “ol” in this context. (Suppressing further errors from this subtree.)
```

Reply:
@dontcallmedom came to the rescue